### PR TITLE
[BAQE-545] Remove deployment scaling in scenario initialization phase

### PR DESCRIPTION
**Pull request by Sgitario**

### CODE REVIEW

File: framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/builder/ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl() {
 
         ProjectSpecificPropertyNames propertyNames = ProjectSpecificPropertyNames.create();
         envVariables.put(propertyNames.workbenchHttpsSecret(), OpenShiftConstants.getKieApplicationSecretName());
-        envVariables.put(propertyNames.workbenchMemoryLimit(), "4Gi"); // limit memory limit to use only 4Gi
+        envVariables.put(propertyNames.workbenchMemoryLimit(), "2Gi"); // limit memory limit to use only 4Gi
```
Comment (Sgitario) [inline framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/builder/ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java]: According to my testing, it seems to be working fine with 2Gi and with this constraint I guess tests will require less to work. However, let me know if you want it to keep it with 4Gi.

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/ldap/WorkbenchKieServerPersistentScenarioLdapIntegrationTest.java [hunk 2]
```diff
@@ -104,24 +104,24 @@ public static void initializeDeployment() {
...
+
 
     @AfterClass
     public static void cleanEnvironment() {
         ScenarioDeployer.undeployScenario(deploymentScenario);
     }
 
     @Test
-    @Ignore("Ignored as the tests are affected by RHPAM-1354. Unignore when the JIRA will be fixed. https://issues.jboss.org/browse/RHPAM-1354")
+    // @Ignore("Ignored as the tests are affected by RHPAM-1354. Unignore when the JIRA will be fixed. https://issues.jboss.org/browse/RHPAM-1354")
```
Comment (sutaakar) [inline test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/ldap/WorkbenchKieServerPersistentScenarioLdapIntegrationTest.java]: The line can be deleted, if the issue is fixed and test is working.

File: framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/builder/ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java (hunk 1 above)
Comment (sutaakar) [reply framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/builder/ClusteredWorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java]: If it works with 2Gi then it is ok.

File: test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/ldap/WorkbenchKieServerPersistentScenarioLdapIntegrationTest.java (hunk 2 above)
Comment (Sgitario) [reply test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/ldap/WorkbenchKieServerPersistentScenarioLdapIntegrationTest.java]: Sorry. It has to be ignored back. Fixed.

**Pull request merged**